src/components/header/template.njk

```diff
@@ -69,7 +69,7 @@
       {% if params.navigation | length %}
       <nav class="govuk-header__navigation" aria-label="Top Level Navigation">
         <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
-        <ul id="navigation" class="govuk-header__navigation-list {{ params.navigationClasses if params.navigationClasses }}">
+        <ul id="navigation" class="govuk-header__navigation-list{{ ' ' + params.navigationClasses if params.navigationClasses }}">
```

Comment:
@JoPintoPaul I'm not sure if nunjucks is ok with these parenthesis but I find this easier to read:
```suggestion
        <ul id="navigation" class="govuk-header__navigation-list{{ (' ' + params.navigationClasses) if params.navigationClasses }}">
```